test/Microsoft.AspNetCore.Razor.Test/TestFiles/CodeGenerator/Output/PrefixedAttributeTagHelpers.Reversed.cs

```diff
@@ -220,15 +198,11 @@ public PrefixedAttributeTagHelpers()
             {
                 throw new InvalidOperationException(FormatInvalidIndexerAssignment("string-prefix-thyme", "TestNamespace.InputTagHelper1", "StringDictionaryProperty"));
             }
-#line 22 "PrefixedAttributeTagHelpers.cshtml"
-__TestNamespace_InputTagHelper2.IntDictionaryProperty["value"] = 37;
-
-#line default
-#line hidden
-            __tagHelperExecutionContext.AddTagHelperAttribute("int-prefix-value", __TestNamespace_InputTagHelper2.IntDictionaryProperty["value"]);
+            __TestNamespace_InputTagHelper2.IntDictionaryProperty["value"] = __tagHelperAttribute_12.Value.ToString();
```

Comment:
This is not equivalent to what was there before. It's ToString()ing an int when the value should actually be an int